src/contentmanager.cpp

```diff
@@ -86,6 +85,12 @@ QStringList ContentManager::getBookInfos(QString id, const QStringList &keys)
         if (key == "mediaCount") {
             values.append(QString::number(b->getMediaCount()));
         }
+        if (key == "tags") {
+            QStringList tagList = QString::fromStdString(b->getTags()).split(';');
+            tagList.replaceInStrings(QRegExp("^_.*$"), "");
```

Comment:
`filter` would have been better here. Because this is what we do : Remove tags starting by `_`.

Replace the tag by an empty string :
- Is more complex to understand what we do.
- Add an extra space around the "hidden" tag.